### DISCUSSION TITLE
custom login problem

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#173] [General]
i have configured my the hfs with first page that is custom page html. the user click on link in this page, its opens dialog login. the user  logons successful, and he redirects on location. if user on address  'http://192.168.99.247/sektor1/deljeni', removes '/sektor1/deljeni', its opens the first page. the user clicks again on link its opens page '/login' instead the dialog login, because session is active. the same problem is if the user clicks on x button on window of web browser

**Comment by gogi100** [comment]
the configuration of folder on my hfs for vfs virtual 'login'
![login](https://user-images.githubusercontent.com/21057856/227163829-ae59e843-1071-4429-8893-19b5db96a999.jpg)

**Comment by rejetto** [comment]
So, this is not a bug, but it's also not what you want.
You want people to get login dialog when they are at /login.
Of course the name "login" here means nothing to HFS, it's just a folder.
I will think of a possible feature to get what you want.

**Comment by gogi100** [comment]
i use the 'login' folder that i set link <a href="/login">Пријавите се</a> on the my first page because i cannot set on dialog login

**Comment by rejetto** [comment]
ok, let's see if this solution suits you better.
1. Use [this preview version](https://github.com/rejetto/hfs/files/11056247/hfs-windows.zip)
2. set the link in your html to /login#LOGIN

Let me know.

If you want, you can also try the csv: Admin > accounts > add > from csv

**Reply by gogi100** [reply]
i set in link of my first page '/login#LOGIN, when  the user types the credentials, redirecting on folder  its'not works, just stands like on picture
![redirecting](https://user-images.githubusercontent.com/21057856/227455382-5b542ff0-7744-4931-a24e-db399a980c08.jpg)

**Comment by rejetto** [comment]
look in the browser console if there is any error
(F12 > console)

**Reply by gogi100** [reply]
i used F12>console
![developying](https://user-images.githubusercontent.com/21057856/227472841-7659e1cb-b055-418c-8c58-832ac18fa688.jpg)

**Comment by rejetto** [comment]
i think i fixed the bug, try this: 
[hfs-windows.zip](https://github.com/rejetto/hfs/files/11060752/hfs-windows.zip)

**Reply by gogi100** [reply]
now its works but option for access admin panel, when i set 192.168.99.247 i cannot access, maybe set disable access address that i set in 
![access admin panel](https://user-images.githubusercontent.com/21057856/227501173-778ad110-07d8-4ca4-b0f5-53e3ec455bfc.jpg)

**Comment by rejetto** [comment]
is 192.168.99.247 ip of the browser computer?

**Comment by gogi100** [comment]
This ip address is computer where is started the hfs

**Comment by rejetto** [comment]
You have to enter the IP of the client computer

**Reply by gogi100** [reply]
i need a rule 'all disabled except on local computer where is started the hfs'

**Reply by rejetto** [reply]
local computer = localhost = 127.0.0.1
so, enter this: 127.0.0.1|::1

i included ::1 that is localhost in IPv6
then you access admin only on http://localhost/~/admin

**Comment by gogi100** [comment]
yes, i helped to me. many thanks. i created issues the theme for blocking extensions and i have one more theme that i will public later topic related to translation of the hfs
